Rubberduck.Resources/Inspections/InspectionNames.fr.resx

```diff
@@ -393,7 +393,7 @@
   <data name="ObjectWhereProcedureIsRequiredInspection" xml:space="preserve">
     <value>Un appel de procédure est attendu, mais un objet est fourni</value>
   </data>
-  <data name="IndexedDefaultMemberAccessInspection" xml:space="preserve">
```

Comment:
This was probably a typo, but it's failing the build right now, as the test that makes sure all inspections have a key in InspectionNames is now failing for `IndexedDefaultMemberAccessInspection`.
The resource key needs to be reinstated everywhere it was accidentally renamed, for the build to pass.